packages/integration-tests/index.js

```diff
@@ -258,8 +258,8 @@ async function addProviders(stackName, bucketName, dataDirectory, s3Host = null)
  * @returns {Promise.<integer>} number of rules added
  */
 async function addRules(config, dataDirectory) {
-  const { stackName, bucketName } = config;
-  const rules = await setupSeedData(stackName, bucketName, dataDirectory);
+  const { stackName, bucket } = config;
```

Comment:
I think we should probably make this consistent for all bucketName/bucket variables that at their root come from the testUtils.loadConfig  (config.bucket) for consistency (e.g. in addCollection, addProvider), and probably make the signatures consistent between this method and  addProviders/addCollections as well.

Reply:
Yeah,  def for the signature.  I think I had good reasons for the bucket/bucketName when I did it, but bucketName matches the rest of this file.  I'll change that too.

Reply:
Well, actually.  The config is used to template the rules.json files because we need to replace AWS region and accountID before uploading to the DB.  So I think we need to pass in the whole config for adding rules.  

Weirdly, there's pseudoTemplating because addProviders overrides the `host` key only in the special case of an S3 protocol . https://github.com/nasa/cumulus/pull/455/files#diff-c758a8c9441f858914c13f40f7b9a94fR244 .  

I don't know if it's possible or necessary to unify the signatures and templating of the json files..

Reply:
👍

Reply:
Actually we can't pull bucketName out of config, and need the bucket object from config.

Reply:
Hmm, indeed.  We could reassign it to be ... more clear, but that doesn't really feel useful.       The issue (for posterity) is the key in the config doesn't match what we're calling it in the code because the other functions  are ... translating, whereas addRules is pulling the value directly.     This is a level of pedantry that isn't helpful in this PR, but I might take a stab at a refactor elsewhere - I'll drop the concern here.    Thanks for thinking about it.